### ISSUE TITLE
IK+ broken audio (WHDLoad 1.8 slave)

### ISSUE THREAD

**Issue by RetroD** [midwan/amiberry#256]
### Expected behavior
All sound effects playing like in the ADF version
### Actual behavior
Some sound effects of the characters are either not playing or are cut out prematurely 
### Steps to reproduce the behavior
(tested on Retropie and Amiberry 2.13)
Use the 1.8 version (same results with version 1.7) of the WHDLoad slave for IK+.
Run the game using WHDLoad autoboot program and the auto generated uae file from the UAE config maker.
As you play the game you will notice some sound effects are missing or being cut out. Note that the same slave runs without issues on FS-UAE on windows. Also note that the adf version running on amiberry has no sound issues.

I know that a previous version of the slave (1.5) is known to have sound issues (this was also the case on FS-UAE, and as far as I know also on a real Amiga). The slave has since been updated to address these issues but this latest version has more sound issues on Amiberry and I couldn't find any settings that would fix it.

**Comment by RetroD** [comment]
After testing this the whole night I gave up, turned off my rpi. Then turned it on a bit later to try something else I was reading up online and the issues were gone! (without me making any changes)

Its weird, I updated the game slave from 1.5 to 1.8 and I was having the issues, and they went away with a restart of the pi. Not sure if there's anything cashed somewhere by WHDLoad, but it looks like it was a simple turn off/on solution, and I can't believe I spent my whole night on that for nothing. sorry for opening this for no reason.
